_posts/1970-01-01-architecture.md

```diff
@@ -156,9 +156,8 @@ TODO: ROM example
 
 Is a collection of next-generation Ruby libraries, each intended to encapsulate a common task. Decoupled and reusable.
 
-TODO: Add extended dry-rb gems description
```

Comment:
Please, can you leave this TODO, as I really want to put here a list of brief descriptions and examples for most notable dry gems (as I see them): validation, types, container, auto_inject, component, configurable.